ui/app/adapters/capabilities.js

```diff
@@ -44,5 +44,18 @@ export default ApplicationAdapter.extend({
       resp.path = id;
       return resp;
     });
-  },
-});
+  }
+
+  query(store, type, query) {
+    const paths = query?.paths.map((p) => this._formatPath(p));
+    return this.ajax(this.buildURL(type), 'POST', {
+      data: { paths },
+      namespace: sanitizePath(this.namespaceService.userRootNamespace),
```

Comment:
Interesting that we need this 🤔 can you explain a bit about why you added it?

Reply:
I copied it from the method above 🤷

Reply:
I was hoping you could tell me why it was there 😅  https://github.com/hashicorp/vault/issues/24168 

if we need it for query, it's probably something that should be added to each model, and not at the top level here

Reply:
I think I can go ahead and remove it, there isn't a namespace coming in when I check the serializer for it.

Reply:
For posterity, we discussed on slack and we **do** need this, added test coverage around it.